### ISSUE TITLE
5 stars question

### ISSUE THREAD

**Issue by umito** [microsoft/rate-my-app#20]
I really dislike the suggestion to the user to rate the app 5 stars. It feels spammy and not sincerely. Why isn't chosen to just ask the user for a rating, whatever that rating may be? That would make feedback more honest and interesting.

**Comment by ltomuta** [comment]
The idea is that what you as an app author want is not just rating, but good rating. This would reflect the quality of the app and thus help you sell the app better. If the user does not agree that your app deserves 5 stars then the user will decline to rate the app and in the next prompt [s]he will be prompted to give you feedback, which will then help you improve the app to the point of deserving the 5 starts in one of the later releases.

Does this make sense?

**Comment by umito** [comment]
Somewhat, but still this inflates the market place rating system by motivating only to leave feedback when a user wants to rate 5 stars. 

I could not test the control on a device, only on emulator, so I don't know if it's possible for the user to also give 4 stars if he wants, or that it's all or nothing.

**Comment by jaaura** [comment]
User can give any rating.

**Comment by umito** [comment]
Ok, then it's just my personal preference of disliking to 'influence the users give a 5 star rating.
Glad to see the user can just rate what he wants.

**Comment by hamishwillee** [comment]
For what it is worth, I 100% agree with umito. It may be cultural - but this made me think less of the app. I also think that if I start seeing that posted message "This is an awesome app, I use it every day" (or whatever it was) I will start to ignore it. I don't have a solution, but I do see it as a problem.

**Comment by ltomuta** [comment]
How about this: change the button to "Rate me" and the body of text above it into something like this: "We would appreciate a good rating in the store, it would help us attract more users. Please give us a good rating by tapping "Rate me" or if you think we should be improving the app, tap on "No thanks" and then send us your feedback"

That's an awfully long text, likely to get worse in German. Any suggestions for a better string?

Or, a "rate me" alternative to "rate 5 stars" which could be selected via a book property like DriveHighRating

If (DriveHighRating)
     Use5StarString()
Else
     UseRateMeString()
Endif

Sent from my Nokia Lumia 920 powered by Windows Phone 8

**Comment by hamishwillee** [comment]
There is no panacea. How about:
Select "Rate me" if you like the app or "No thanks" to send improvement ideas and feedback.

**Comment by umito** [comment]
Yes, best would indeed just to remove the '5 stars" part of the message, and keep everything else as it is.
So then you would get: 'We'd love you to rate our app." and the 'rate 5 stars' message to become 'rate' or as suggested above 'Rate me', or to keep in the plural sense of the whole tool: 'Rate us" or "Rate {0}"
